src/guess_density.jl

```diff
@@ -1,6 +1,9 @@
-
 """
+    random_density(basis; n_electrons=basis.model.n_electrons)
+
 Generate a physically valid random density integrating to the given number of electrons.
+If the basis is has more than one spin component, the spin density is randomly generated
+without symmetry.
 """
 function random_density(basis::PlaneWaveBasis; n_electrons=basis.model.n_electrons)
```

Comment:
I wonder if this should become a `guess_density(basis, RandomDensity(n_electrons))`.